debezium-connector-mysql/src/test/resources/ddl/mysql-test-statements.ddl

```diff
@@ -271,21 +271,21 @@ CREATE INDEX id_index USING BTREE ON lookup (id);
 
 -- CREATE [ONLINE|OFFLINE] [UNIQUE|FULLTEXT|SPATIAL] INDEX index_name
 
-CREATE ONLINE INDEX index_1;
-
-CREATE OFFLINE INDEX index_2;
-
-CREATE ONLINE UNIQUE INDEX index_3;
-
-CREATE OFFLINE FULLTEXT INDEX index_4;
-
-CREATE UNIQUE INDEX index_5;
-
-CREATE FULLTEXT INDEX index_6;
-
-CREATE SPATIAL INDEX index_7;
-
---CREATE LOGFILE GROUP lf_group_name
+# CREATE ONLINE INDEX index_1;
```

Comment:
Why is this commented?

Reply:
There is no existence of ONLINE INDEX in MySQL, only UNIQUE,FULLTEXT,SPATIAL.